src/orka-builder.ts

```diff
@@ -70,7 +70,9 @@ export default class OrkaBuilder {
     );
     this.useCors();
     this.use(() => addRequestId(this.config));
-    this.use(() => addVisitorId(this.config));
+    if (includeVisitor) {
```

Comment:
@nikostoulas shall we remove it from the defaults altogether? Is anyone depending on it.

Otherwise, @klesgidis we should switch to use a config for this one, instead of an argument. How about `config.visitor.no_orka` or something like that.

Reply:
The middleware itself does the below check:   
 ```
if (ctx.path === '/health' || !config.visitor || !config.visitor.cookie) {
```

We could remove it from there and add it here. 
If config.visitor.cookie
If for some reason this doesn't work for you we could also use the config.visitor.enabled which is more explicit.

Reply:
I think no-one else is using it. But making it easy to be used (eg with no code changes only an env config) would be nice.

Reply:
Let's go with either `config.visitor.enabled` or renaming the env variable `config.visitor.cookieName` ? I believe the middleware was forked from our codebase, so we ended up parsing the cookie twice, and the code itself is not inline now.